README.md

```diff
@@ -15,9 +15,10 @@ This will configure the target `census-eq` to point to the eq team on Concourse.
 ## Development Environments
 
 #### Prerequisites
-You must have a project with a *Google Container Registry (GCR)* already set up to be able to push built images.
-Set your registry in *variables.yaml* to `eu.gcr.io/<gcp_project_id>`. The `gcp_project_id` should be the ID of the GCP project under which the container registry is hosted.
-Ensure in the GCR settings, the visibility is set to Public. To do this, you must first initialize a registry manually via the UI or run the docker build step which would fail the first time since it would be private.
+- An existing GCP project.
+- *Google Container Registry (GCR)* enabled for the GCP project.
+- Set the registry in *variables.yaml* to `eu.gcr.io/<gcp_project_id>`. The `gcp_project_id` should be the ID of the GCP project under which the container registry is hosted.
+- Ensure the visibility of the GCR is set to Public. To do this, you must first initialize a registry manually via the UI or run the docker build step (which will fail on the first run due to not having initialized the registry).
```

Comment:
Currently, the docker build step doesn't exist.

Reply:
See above comment